docs/Authoring.md

```diff
@@ -125,6 +125,23 @@ directory by using `Link` metadata to specify file path in output package.
 </ItemGroup>
 ```
 
+To package content v2 files, use `ContentFile` item type. This item type supports 
+`PackageBuildAction`, `PackageCopyToOutput` and `PackageFlatten` metadata. 
```

Comment:
In your sample below, the metadata is `ContentFileBuildAction`, which doesn't appear in this list.
I wonder if it should be `ItemType` if its purpose is to identify the item type that it is injected as into the receiving project.